src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -207,7 +204,15 @@ private async Task LoadOrReloadProjectAsync(string projectPath, ProjectBuildMana
                 {
                     foreach (var logItem in loadedFile.Log)
                     {
-                        _logger.LogWarning($"{logItem.Kind} while loading {logItem.ProjectFilePath}: {logItem.Message}");
+                        if (logItem.Kind is WorkspaceDiagnosticKind.Failure)
+                        {
+                            _logger.LogError($"{logItem.Kind} while loading {logItem.ProjectFilePath}: {logItem.Message}");
```

Comment:
There are extension methods for ILogger that will just take the severity as a parameter, so you don't have to duplicate the message logic here:

https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.log?view=dotnet-plat-ext-7.0#microsoft-extensions-logging-loggerextensions-log(microsoft-extensions-logging-ilogger-microsoft-extensions-logging-loglevel-system-string-system-object())